### PR TITLE
Patch geno one extensions bug

### PR DESCRIPTION
**Pull request by stschiff**
Alright. I think I fixed it, and also added tests.

### CODE REVIEW

File: poseidon-hs.cabal [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:                poseidon-hs
-version:             1.5.7.0
+version:             1.5.7.1
```
Comment (nevrome) [inline poseidon-hs.cabal:2]: I took v1.5.7.1 already for #313. Wer zuerst kommt patched zuerst :grimacing:

File: poseidon-hs.cabal [hunk 2]
```diff
@@ -58,7 +58,8 @@ Test-Suite poseidon-tools-tests
                         Poseidon.SummariseSpec, Poseidon.SurveySpec, Poseidon.GenotypeDataSpec,
                         Poseidon.EntitiesListSpec, PoseidonGoldenTests.GoldenTestsValidateChecksumsSpec,
                         PoseidonGoldenTests.GoldenTestsRunCommands, Poseidon.ChronicleSpec,
-                        Poseidon.SequencingSourceSpec, Poseidon.InterfaceSpec
+                        Poseidon.SequencingSourceSpec, Poseidon.InterfaceSpec,
```
Comment (nevrome) [inline poseidon-hs.cabal:61]: I think `InterfaceSpec` was specifically meant as `OptParseApplicativeParsersSpec`. We only need one module to test code form `OptparseApplicativeParsers.hs`, right? I say: You choose which one you prefer and we remove the other one.

**Review by nevrome — APPROVED** [review]
Looks good to me. Sad what we need this complexity, but I don't see how else to deal with with multiple dots in the file names. Thanks for writing the informative tests!

File: poseidon-hs.cabal (hunk 1 above)
Comment (stschiff) [reply poseidon-hs.cabal:2]: OK true. I will take whatever comes next.

File: poseidon-hs.cabal (hunk 2 above)
Comment (stschiff) [reply poseidon-hs.cabal:61]: True, thanks for pointing that out. I will move this over, or merge.

**Pull request merged**